pkg/sql/show_tenant.go

```diff
@@ -91,12 +94,36 @@ func (n *showTenantNode) startExec(params runParams) error {
 	if err != nil {
 		return err
 	}
+	var job *jobs.Job
 	n.tenantInfo = tenantRecord
 	jobId := n.tenantInfo.TenantReplicationJobID
-	registry := params.p.execCfg.JobRegistry
-	job, err := registry.LoadJobWithTxn(params.ctx, jobId, params.p.Txn())
-	if err != nil {
-		return err
+	if jobId == 0 {
+		n.replicationStatus = n.tenantInfo.State.String()
+	} else {
+		registry := params.p.execCfg.JobRegistry
+		job, err = registry.LoadJobWithTxn(params.ctx, jobId, params.p.Txn())
+		if err != nil {
+			return err
+		}
+		switch n.tenantInfo.State {
```

Comment:
Overall I like this approach, but I wonder if we are providing too much of a window into our internal job state to users. PauseRequested and Pending are transient states that the job goes through in our state machine. Do we think we want to explain to the end user what these states really mean?

Reply:
WDYT about mapping:
StatusPauseRequested to REPLICATING because if pause was only requested then we are still replicating.
StatusPending to ADD because we just started the replication.

Reply:
done, hopefully this makes sense.
btw I'll add datadriven tests for those states the minute you merge you pr!

Reply:
I like this! and yes getting that PR in is first on my list, it keeps running into issues